lib/account/pages/about_page.dart

```diff
@@ -9,15 +12,101 @@ class AboutPage extends StatefulWidget {
+      ),
+    );
+  }
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
         title: Text(S.of(context).pageAboutTitle),
       ),
       body: Center(
-        child: SingleChildScrollView(
-          child: Text(S.of(context).appName),
+        child: CustomScrollView(
+          physics: const ClampingScrollPhysics(),
+          slivers: [
+            SliverFillRemaining(
+              hasScrollBody: false,
+              child: Padding(
+                padding: const EdgeInsets.all(10),
+                child: Column(children: [
+                  Expanded(child: Container()),
+                  Text(
+                    S.of(context).appName,
+                    style: Theme.of(context).primaryTextTheme.displaySmall,
+                  ),
+                  if (_packageInfo != null) ...[
+                    Text(
+                      S.of(context).pageAboutVersion(_packageInfo!.version),
+                      style: Theme.of(context)
+                          .primaryTextTheme
+                          .subtitle2!
+                          .copyWith(color: Theme.of(context).colorScheme.onSurface.withOpacity(0.5)),
+                      textAlign: TextAlign.center,
+                    ),
+                    Text(
+                      S.of(context).pageAboutBuildNumber(_packageInfo!.buildNumber),
+                      style: Theme.of(context)
+                          .primaryTextTheme
+                          .subtitle2!
+                          .copyWith(color: Theme.of(context).colorScheme.onSurface.withOpacity(0.5)),
+                      textAlign: TextAlign.center,
+                    ),
+                    Text(
+                      S.of(context).pageAboutBuildSignature(_packageInfo!.buildSignature),
+                      style: Theme.of(context)
+                          .primaryTextTheme
+                          .subtitle2!
+                          .copyWith(color: Theme.of(context).colorScheme.onSurface.withOpacity(0.5)),
+                      textAlign: TextAlign.center,
+                    ),
+                  ],
+                  SvgPicture.asset(
+                    'assets/logo.svg',
+                    width: MediaQuery.of(context).size.width / 2,
+                    height: MediaQuery.of(context).size.width / 2,
+                    semanticsLabel: 'MOTIS Logo',
+                  ),
+                  Text(
+                    S.of(context).pageAboutText,
+                    textAlign: TextAlign.center,
+                  ),
+                  Expanded(
+                    child: Align(
+                      alignment: Alignment.bottomCenter,
+                      child: Row(mainAxisAlignment: MainAxisAlignment.center, children: [
+                        TextButton(
+                          onPressed: () async => await launchUrlString(
+                            'https://github.com/MOTIS-Mitfahr-App/flutter-app',
```

Comment:
it would be better to move these links into constants at the top of the file/class